src/services/EdFi.DataManagementService.Api/Infrastructure/WebApplicationBuilderExtensions.cs

```diff
@@ -0,0 +1,19 @@
+// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using EdFi.DataManagementService.Api.Configuration;
+
+namespace EdFi.DataManagementService.Api.Infrastructure
+{
+    public static class WebApplicationBuilderExtensions
+    {
+        public static void AddServices(this WebApplicationBuilder webAppBuilder)
+        {
+            webAppBuilder.Services.Configure<AppSettings>(webAppBuilder.Configuration.GetSection("AppSettings"));
```

Comment:
I like how this is parsing directly into our class without having to write a bunch of cast statements from strings.